frontend/src/components/Reports/Routine.js

```diff
@@ -53,10 +53,17 @@ export const RoutineReportsMenu = {
       icon: IbmWatsonNaturalLanguageUnderstanding,
       SideNavMenuItem: [
         {
-          link:
-            config.serverBaseUrl +
-            "/Report?type=indicator&report=activityReportByTest",
-          label: <FormattedMessage id="sideNav.label.bytesttype" />,
+
+          link:"/RoutineReport?type=indicator&report=activityReportByTest",
+          label: <FormattedMessage id="sideNav.label.bytesttype"/>,
+        },
+        {
+          link:"/RoutineReport?type=indicator&report=activityReportByPanel",
```

Comment:
why ddint you just rename the existing [menu items](https://github.com/I-TECH-UW/OpenELIS-Global-2/blob/develop_3x/frontend/src/components/Reports/Routine.js#L61-L74)??